src/network/utils.ts

```diff
@@ -29,6 +29,13 @@ function isHost(host: any): host is Host {
   return isIPv4 || isIPv6;
 }
 
+function isHostWildcard(host: Host): boolean {
+  const [isIPv4] = Validator.isValidIPv4String(host);
+  if (isIPv4 && host === '0.0.0.0') return true;
+  const [isIPv6] = Validator.isValidIPv6String(host);
+  return isIPv6 && host === '::';
+}
+
```

Comment:
Why not just do `host === '0.0.0.0' || host === '::'`?

Reply:
That works too. I'll fix it up.

Reply:
Pushed up the change.